controller/raft/mesh/mesh.go

```diff
@@ -230,6 +246,12 @@ func (self *impl) AddPeer(peer *Peer) {
 	self.lock.Lock()
 	defer self.lock.Unlock()
 	self.Peers[peer.Address] = peer
```

Comment:
Comment is misleading, not checking for higher, just equality

Reply:
Yea that makes sense. As implemented it would also readonly if an older version joined